bootstrap/017_query_history.sql

```diff
@@ -166,3 +170,32 @@ BEGIN
         call internal.set_config(:key, 'true');
     end if;
 END;
+
+create or replace procedure internal.update_qtag_day()
+returns varchar
+language sql
+comment = 'materialize qtag_filter for a single day until fully backfilled'
+as
+begin
+
+    let res resultset := (
+        SELECT
+            distinct start_time::date as day
+        FROM
+            internal_reporting_mv.query_history_complete_and_daily
+        WHERE
+            qtag_filter IS NULL
+            AND qtag IS NOT NULL
+        ORDER BY day desc
+    );
+    let cur cursor for res;
+    for r in cur do
```

Comment:
I'd put a note here as to why we're doing this a day at a time wrt locking.

Reply:
done